packages/authenticator/amplify_authenticator/lib/src/utils/responsive.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+bool isMobileOrWebMobile(BuildContext context) {
```

Comment:
I might add a doc comment to this. "Mobile" is a little misleading because it might imply that this returns false for tablets, but it returns true. I don't have a better suggestion for the name, but a doc comment would be helpful.